mmdet/core/export/pytorch2onnx.py

```diff
@@ -102,11 +102,13 @@ def build_model_from_cfg(config_path, checkpoint_path, cfg_options=None):
     return model
 
 
-def preprocess_example_input(input_config):
+def preprocess_example_input(input_config, device=torch.device('cpu')):
```

Comment:
Have you tested exporting to ONNX with `device=cuda`?

Reply:
I will check it.

Reply:
It looks like everything works correctly with `device=torch.device('cuda:0')`.